gcb/test/helm/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+#### SECURITY NOTICE ####
+# Google Cloud Build (GCB) supports the usage of secrets for build requests.
+# Secrets appear within GCB configs as base64-encoded strings.
+# These secrets are GCP Cloud KMS-encrypted and cannot be decrypted by any human or system
+# outside of GCP Cloud KMS for the GCP project this encrypted resource was created for.
+# Seeing the base64-encoded encrypted blob here is not a security event for the project.
```

Comment:
Thanks for this notice 👍🏼 I obviously skimmed past  these comments, saw the secret and was very alerted 😄